app/Charts/RatingBarChart.php

```diff
@@ -8,13 +8,27 @@ class RatingBarChart
 {
   use ChartPresenter;
 
-  protected $datasetName = 'Number of User';
-  protected $values = [];
+  protected $datasetName;
+  protected $values;
+  protected $count;
+
+  public function __construct($datasetName = 'Number of User', $values = [], $count = true)
+  {
+    $this->datasetName = $datasetName;
+    $this->values = $values;
+    $this->count = $count;
+  }
 
   public function makeChart($ratings = [], $maxScore = 5)
   {
-    for($score = 1; $score <= $maxScore; $score++){
-      array_push($this->values, $ratings->where('score', $score)->count());
+    if($this->count){
+      for($score = 1; $score <= $maxScore; $score++){
+        array_push($this->values, $ratings->where('score', $score)->count());
+      }
+    } else {
+      foreach($ratings as $rating){
+        array_push($this->values, $rating->score);
+      }
```

Comment:
This method is somewhat confusing to me. It looks like we are making two charts, but it's not clear to me what the difference is based on the condition name (`count`).

I wonder if it'll be more clear if we turn it into two methods or two classes?